### PR TITLE
[CMSSW_12_0_X] Add new Run3 offline GT in autoCond and its relval workflow

### PR DESCRIPTION
**Pull request by tvami**
#### PR description:

Backport of https://github.com/cms-sw/cmssw/pull/35202

### CODE REVIEW

**Comment by tvami** [comment]
backport of https://github.com/cms-sw/cmssw/pull/35202

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
@smuzaffar will this new RelVal input buffering, that you mentioned in https://github.com/cms-sw/cmssw/pull/35202#issuecomment-915927319 , have to happen separately for each branch?

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by qliphy** [comment]
@tvami 

From  https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-a5b48c/18508/dasqueries/run.txt
I see "KeyError: 'RunCosmics2021'"

It seems you should also backport https://github.com/cms-sw/cmssw/pull/34643

**Comment by tvami** [comment]
> @tvami
> 
> From https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-a5b48c/18508/dasqueries/run.txt
> I see "KeyError: 'RunCosmics2021'"
> 
> It seems you should also backport #34643

Thanks!

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by smuzaffar** [comment]
> @smuzaffar will this new RelVal input buffering, that you mentioned in [#35202 (comment)](https://github.com/cms-sw/cmssw/pull/35202#issuecomment-915927319) , have to happen separately for each branch?

yes files should be cached for all releases

**Comment by tvami** [comment]
@cmsbuild  , please test

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -491,7 +491,11 @@
 steps['RunCosmics2016B']={'INPUT':InputInfo(dataSet='/Cosmics/Run2016B-v1/RAW',label='2016B',run=[272133],events=100000,location='STD')}
 
 ### LS2 - MWGR#5 2020 - CSC, DAQ, DCS, DQM, DT, ECAL, GEM, HCAL, RPC, TCDS, TRACKER, TRG ###
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_steps.py:493]: Could you please correct the comment,
### LS2 - MWGR 2021 - CSC, DAQ, DCS, DQM, DT, ECAL, GEM, HCAL, L1SCOUT, PIXEL, RPC, TCDS, TRACKER, TRG ###

Comment (tvami) [reply Configuration/PyReleaseValidation/python/relval_steps.py]: I'd done that

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by srimanob** [comment]
+Upgrade

PRs run fine with the new 2021 cosmics workflows.

**Comment by tvami** [comment]
+alca

**Comment by tvami** [comment]
Hi @kskovpen @cms-sw/pdmv-l2 would you mind signing this backport as well? Thanks!

**Comment by jordan-martins** [comment]
+1

**Comment by tvami** [comment]
Hi @qliphy @perrotta we have another PR (the 12_0_X version of https://github.com/cms-sw/cmssw/pull/35233) in the pipeline for 12_0_X, would you please merge this asap?

**Comment by tvami** [comment]
urgent

**Comment by perrotta** [comment]
> Hi @qliphy @perrotta we have another PR (the 12_0_X version of #35233) in the pipeline for 12_0_X, would you please merge this asap?

Hi @tvami this can be merged in 12_0_X to allow further backports.
Please notice however that 12_0_1 is currently being built, following a request in that sense from PPD, see #35246

**Comment by perrotta** [comment]
+1
- GT updates for Run3 and technical data taking at P5

**Comment by tvami** [comment]
> > Hi @qliphy @perrotta we have another PR (the 12_0_X version of #35233) in the pipeline for 12_0_X, would you please merge this asap?
> 
> Hi @tvami this can be merged in 12_0_X to allow further backports.
> Please notice however that 12_0_1 is currently being built, following a request in that sense from PPD, see #35246

Oh I guess it would have been nice to have that PR included there. On the other hand, it's not a must, the GT can be specified from the cmsDriver...

**Pull request merged**